common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
The page object is created, but what is actually being viewed? The only "visit" in the setup that I see is 

```
# Add a verified mode to the course
         ModeCreationPage(
             self.browser, self.course_id, mode_slug=u'verified', mode_display_name=u'Verified Certificate',
             min_price=10, suggested_prices='10,20'
         ).visit()

```

Is that sufficient?

It is also my understanding that unless you use specific scoping for the a11y test, you test everything visible on the page. Therefore if these page objects are only regions within the same page (and they are all already visible), there is no need to run an accessibility check on every one-- it is just adding build time. 

That would also explain why they have all the same "ignores".